src/react_native/reanimated.cljs

```diff
@@ -115,13 +118,15 @@
                                              (js-obj "duration" duration
                                                      "easing"   (get easings easing))))))
 
-(defn animate-shared-value-with-delay-default-easing
-  [anim val duration delay]
-  (set-shared-value anim
-                    (with-delay delay
-                                (with-timing val
-                                             (js-obj "duration" duration
-                                                     "easing"   (in-out (.-quad ^js Easing)))))))
+(defn animate-delay
+  ([animation val delay]
+   (animate-delay animation val delay default-duration))
+  ([animation val delay duration]
+   (set-shared-value animation
```

Comment:
Thanks for your review @ilmotta.

Visually, this suggestion looks better. However, I find it more difficult to read in reverse.

Reply:
Fair point! It's just a bit rare to see Clojure code formatted that way. The thread operator is also known as a pipeline macro, since you read from left -> right and top -> bottom. The original implementation forces the reader to read from inside out (which is the order of evaluation).

But it's personal preference most of the time :) and thanks for taking the time to express your opinion.